src/lua/api-gateway/util/OauthClient.lua

```diff
@@ -16,6 +16,52 @@ function OauthClient:new(o)
     return o
 end
 
+--- Loads a lua gracefully. If the module doesn't exist the exception is caught, logged and the execution continues
+-- @param module path to the module to be loaded
+--
+local function loadrequire(module)
+    ngx.log(ngx.DEBUG, "Loading module [" .. tostring(module) .. "]")
```

Comment:
this does not work - it's a mircale it's working - due to the fact that I am changing the ngx.log implementation somewhere above - due to the fact that you are requiring this in a init phase this is not supposed to work in any ngx implementation - please correct this

Reply:
@atrifan the loadrequire method is the same throughout the entire GW project. Are you suggesting we change it everywhere based on these three comments?